cmd/atlas/internal/docker/docker.go

```diff
@@ -131,6 +135,19 @@ func FromURL(u *url.URL) (*Config, error) {
 			opts = append(opts, Env("POSTGRES_DB="+parts[1]))
 		}
 		cfg, err = PostgreSQL(tag, opts...)
+	case DriverMSSQL:
+		if len(parts) > 1 {
+			if db := parts[1]; db != "master" {
+				opts = append(opts, Setup(fmt.Sprintf("CREATE DATABASE [%s]", db)))
+			}
+		}
+		if tag == "" {
+			tag = "2022-latest"
+		}
```

Comment:
Th tag is not required, you can still call `docker pull mcr.microsoft.com/mssql/server` no?